commercial/app/model/commercial/events/LiveEventAgent.scala

```diff
@@ -64,10 +62,12 @@ object LiveEventAgent extends ExecutionContexts with Logging {
 
     val liveEvents =
       for{
-        eventImages <- fetchAndParseLiveEventImages
+        eventMembershipInfo <- fetchAndParseLiveEventsMembershipInfo
         ParsedFeed(events, _) <- Eventbrite.parsePagesOfEvents(feedMetaData, feedContent)
-      } yield events
-            .map ( event => LiveEvent(event, eventImages find (_.eventId == event.id)) )
+      } yield events flatMap { event =>
```

Comment:
Not that I could see; if we can't find a matching image and URL for a given `LiveEvent` then we want to discard it the `LiveEvent` completely.

If I add in `event <- events` to the comprehension, then I wouldn't be able to filter those events out unless I used a guard on the for comprehension. However, in order to use a guard, I'd have to also perform the matching in the comprehension (to then use the result in the guard), which cannot be done since I am iterating over `Futures` and the matching returns an `Option`, and I cannot easily convert between the two.